src/main/java/com/exadel/discountwebapp/location/repository/LocationRepository.java

```diff
@@ -13,4 +14,6 @@
     List<Location> findAllByCountry_CountryCode(String countryCode);
 
     List<Location> findAllByCity(String city);
+
+    List<Location> findAllByIdIn(Collection<Long> ids);
```

Comment:
There is no need to write this method by hand. It already exists in CrudRepository class (findAllById).

Reply:
fixed